pipeline/terraform/main.tf

```diff
@@ -1,8 +1,33 @@
-module "catalogue_pipeline_20201111" {
+locals {
+  stacks = {
+    "20201111" = {
+      release_label     = "prod"
+      enable_reindexing = false
+    }
+
+    # A new pipeline with transformers that include support for multiple languages.
+    # See https://github.com/wellcomecollection/platform/issues/4864
+    "2020-11-12" = {
+      release_label     = "stage"
+      enable_reindexing = true
+    }
+  }
+}
+
+module "stack" {
   source = "./stack"
 
-  pipeline_date = "20201111"
-  release_label = "prod"
+  for_each = local.stacks
+
+  pipeline_date = each.key
+  release_label = each.value.release_label
+
+  enable_reindexing = each.value.enable_reindexing
+
+  enable_sierra_reindexing = lookup(each.value, "enable_sierra_reindexing", false)
```

Comment:
hmmm, I'm kind of on the fence about this - I know that at least @alicefuzier and I sometimes have the reindexer topics enabled and the update topics disabled (as eg it can be useful to test on a pipeline that's not going to have unexpected stuff coming through), and also it's a lot of code to be able to use some booleans... Perhaps we say that it's worth it for the majority of use cases but I'm not sure.

Reply:
[nod]

I haven't done much pipeline work, so maybe the old approach is fine – but the first time I spun up the pipeline I got the "comment/uncomment these topics" thing wrong.

A much less destructive approach I could try would be to move the topic definitions to the top of the file. Right now it's

```
module "stack" {
  // names you need to change
  // stuff that never changes
  // topics you have to change (sometimes)
  // more stuff that never changes
}
```

and so it was easy for me to miss the topics.

Reply:
👍  yup definitely for me it's super useful to be able to disable updates and only have reindexes enabled. It's especially useful when I'm testing something that spans multiple services like merging and I can send specific records from the reidexer

Reply:
👍

I'll defer to your pipeline experience. :)

Reply:
agree that it's diffult to rmember to uncomment/comment the topics. I also made that mistake multiple times